### PR TITLE
Covers Use of Code Generation and Literals for Assets

### PR DESCRIPTION
**Pull request by mliberatore**
## What it Does

Covers the way we apply assets from asset catalogs, specifically to use IB and code gen’d constants and not literals.

### CODE REVIEW

File: Assets.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Assets
-### When do we use them?
-We use asset catalogs to manage our application's assets.
+### How do we use them?
+We use asset catalogs to organize our application’s images and colors.
 
 ### Key Considerations
-* Group assets using folders inside asset catalogs.
+* Organize assets using folders inside asset catalogs.
+* Whenever possible, apply images and colors from asset catalogs using Interface Builder.
```
Comment (mattbischoff) [inline Assets.md]: I quibble with the word "apply". I’d say "use" or "reference"

File: Assets.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Assets
-### When do we use them?
-We use asset catalogs to manage our application's assets.
+### How do we use them?
+We use asset catalogs to organize our application’s images and colors.
```
Comment (mattbischoff) [inline Assets.md]: Any other of the supported media types of asset catalogs we should consider?

Comment (mliberatore) [reply Assets.md]: Good question. I only want to apply specific rules to images and colors at this time as I have in the "Key Considerations", so I’m going to revert "images and colors" to "assets". I don’t want to get into the specifics of how we might interact with the many many things that can go into asset catalogs, beyond the fact that they should exist there, which the original statement covers.

Comment (mattbischoff) [reply Assets.md]: Makes sense to me.

**Pull request merged**